cmd/lotus-shed/migrations.go

```diff
@@ -21,8 +21,8 @@ import (
 )
 
 var migrationsCmd = &cli.Command{
-	Name:        "migrate-nv15",
-	Description: "Run the specified migration",
+	Name:        "migrate-nv16",
```

Comment:
Idea: it might be nice to keep track of the old ones along with brining in the new ones in case someone wants to play with historical migrations.  Probably more work to factor things out right now than its worth 🤷